### PR TITLE
Asset pallet integration

### PR DESCRIPTION
**Pull request by ayushmishra2005**

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -1080,6 +1080,27 @@ impl membership_supernodes::Config for Runtime {
     type Event = Event;
 }
 
+parameter_types! {
```
Comment (ltfschoen) [inline runtime/src/lib.rs:1083]: we're still using Substrate v3.0.0 (see Cargo.toml), not Substrate's latest 'master' branch.
so i'd suggest only adding the Asset pallet that we know is compatible with Substrate v3.0.0
https://github.com/paritytech/substrate/blob/v3.0.0/frame/assets/src/lib.rs#L1209
and list the constants in the same order.

Comment (ltfschoen) [reply runtime/src/lib.rs:1083]: i just tried to git checkout v3.0.0 of Substrate, and the code actually appears the same as what you've done, so i think what you've already done should be sufficient

Comment (ayushmishra2005) [reply runtime/src/lib.rs:1083]: Yeah. But we need to set right parameter for Asset pallet to make it work for our application.

Comment (ltfschoen) [reply runtime/src/lib.rs:1083]: > Yeah. But we need to set right parameter for Asset pallet to make it work for our application.

yes, and bear in mind that we haven't done that for most of the existing parameters for other pallets yet.

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -1145,6 +1166,7 @@ construct_runtime!(
         MiningClaimsHardware: mining_claims_hardware::{Module, Call, Storage, Event<T>},
         MiningExecutionToken: mining_execution_token::{Module, Call, Storage, Event<T>},
         ExchangeRate: exchange_rate::{Module, Call, Storage, Event<T>},
+        Assets: pallet_assets::{Module, Call, Storage, Event<T>},
```
Comment (ltfschoen) [inline runtime/src/lib.rs:1169]: i think this should be listed immediately below Tips instead of at the end

**Review by ltfschoen — COMMENTED** [review]
see comments

File: runtime/src/lib.rs (hunk 2 above)
Comment (ayushmishra2005) [reply runtime/src/lib.rs]: Done

**Comment by ayushmishra2005** [comment]
Closing This pull request for now.

**Pull request closed without merging**